core/src/test/java/tachyon/client/TachyonFSTest.java

```diff
@@ -176,7 +176,6 @@ public void deleteFileTest() throws IOException {
     List<ClientWorkerInfo> workers = sTfs.getWorkersInfo();
     Assert.assertEquals(1, workers.size());
     Assert.assertEquals(WORKER_CAPACITY_BYTES, workers.get(0).getCapacityBytes());
-    Assert.assertEquals(0, workers.get(0).getUsedBytes());
```

Comment:
@haoyuan 
here is my observation:
- master branch, I occasionally got assert failure at this line 
```java
Assert.assertEquals(0, workers.get(0).getUsedBytes());
```

- 0.5 branch, I have some consistent failure at this line
```java
      Assert.assertEquals(writeBytes * (4 - k), workers.get(0).getUsedBytes());
```
This somehow was not seen before, even the code base is the same.

The stack is 
```java
Running tachyon.client.TachyonFSTest
Tests run: 34, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 82.112 sec <<< FAILURE! - in tachyon.client.TachyonFSTest
deleteFileTest(tachyon.client.TachyonFSTest)  Time elapsed: 3.247 sec  <<< FAILURE!
java.lang.AssertionError: expected:<8000> but was:<10000>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:743)
	at org.junit.Assert.assertEquals(Assert.java:118)
	at org.junit.Assert.assertEquals(Assert.java:555)
	at org.junit.Assert.assertEquals(Assert.java:542)
	at tachyon.client.TachyonFSTest.deleteFileTest(TachyonFSTest.java:212)
```